.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "semistandard",
+  "env": {
+    "es6": true,
+    "browser": true,
+    "jasmine": true
+  },
+  "globals": {
```

Comment:
I dont know the code very well but could we use requires instead?

Reply:
$ and Clipboard are added because of the styleguide.js file

Reply:
what about [specifying-globals](https://eslint.org/docs/user-guide/configuring#specifying-globals) in `styleguide.js` file and removing them from the config?

Reply:
It can be done this way